packages/docs/src/pages/patterns/overview.md

```diff
@@ -0,0 +1,15 @@
+---
+label: Overview
+title: Patterns
+weight: -1
```

Comment:
It looks like both the markdown and KSS pages have a default of `weight: 0`. I couldnt figure out how to change the default for both to `1`, so I did this kinda hacky thing instead haha... open to suggestions here

Reply:
I think this is where the default is set to `0` https://github.com/CMSgov/design-system/blob/5675f0b8c1e6bf297bd71a37d539bc0753916d9c/tools/gulp/docs/processMarkdownPage.js#L64

Reply:
Yeah that was the default for the markdown pages but I couldnt seem to find where the default is set for KSS.